pkg/destinations/bigquery/bigquery.go

```diff
@@ -0,0 +1,64 @@
+package bigquery
+
+import (
+	"context"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/rs/zerolog/log"
+	"github.com/scratchdata/scratchdata/util"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+type BigQueryServer struct {
+	DatasetID             string `mapstructure:"bigquery_dataset_id"`
+	CredentialsJsonString string `mapstructure:"bigquery_credentials_json"`
```

Comment:
```suggestion
	CredentialsJsonString string `mapstructure:"credentials_json"`
```

The credentials are used for both GCS and BigQuery